support/docker/nerves_system_br/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM hexpm/erlang:24.2-ubuntu-focal-20210325
+FROM hexpm/erlang:24.2.1-ubuntu-focal-20210325
```

```suggestion
FROM hexpm/erlang:24.2.1-ubuntu-impish-20211102
```
Is there a particular reason to stay on `focal` vs the newer `impish`? If so, then `24.2.1-ubuntu-focal-20211006` is the tag you'll need here

---

O good point. I totally didn't notice it wasn't LTS